webapp/src/Entity/ContestProblem.php

```diff
@@ -46,12 +46,13 @@ class ContestProblem
     private string $shortname;
 
     /**
-     * @ORM\Column(type="integer", name="points", length=4,
+     * @ORM\Column(type="integer", name="points",
```

Comment:
This will result in a migration update if we would run `webapp/bin/console doctrine:migrations:diff`. Why do we want this?

Reply:
Its a bit ugly but:

login here: admin/admin
https://www.domjudge.org/demoweb/jury/contests/7/edit

Search for: #contest_problems_0_points
see the attribute maxlength which is not valid for input['number']

I think the migration was not needed here but I might have overlooked this, but as it works in CI I think this is also ignored for doctrine when using integers?

Reply:
afaik length is used for integers, it makes it into a `int(4)` in the database, but I might be wrong.
Migrations aren't enforced, so it makes sense CI doesn't complain.
Also we have `length=x` for many many integers in our entity, so if Doctirne indeed ignores it (which I think it doesn't) we need to get rid of them all.

Reply:
Update: adding `, IntegerType::class` to the builder in `ContestProblemType.php` is the better fix

Reply:
And is now committed